src/auth.py

```diff
@@ -25,13 +32,17 @@ def get_mediafile_id(meeting_id, path, app, cookie):
         )
 
     try:
-        id = int(response.json()["id"])
+        id = int(response.json()[0])
     except Exception:
         raise ServerError("The Response did not contain a valid id.")
     return id
 
 
-def get_check_request_url():
-    presenter_host = "todo"
-    presenter_port = "todo"
+def get_presenter_url(meeting_id, path):
+    presenter_host = os.environ.get("PRESENTER_HOST")
+    presenter_port = os.environ.get("PRESENTER_PORT")
+    if presenter_host is None:
+        raise ServerError("PRESENTER_HOST is not set")
```

Comment:
Can you make this check early on startup? If the service is misconfigured, it should fail (exitcode >0) early and not on the first request. Maybe create a module, that handles all configuration variables in one place. The same should be done for the postgres configuration.

Reply:
Add the env options "PRESENTER_HOST" and "PRESENTER_PORT" to the config options and check them at start, also fix the config check mechanism.